model/project.go

```diff
@@ -698,6 +698,11 @@ func NewTaskIdTable(p *Project, v *Version, sourceRev, defID string) TaskIdConfi
 
 	sort.Stable(p.BuildVariants)
 
+	projName := p.Identifier
+	name, err := GetIdentifierForProject(p.Identifier)
```

Comment:
can you rename these variables? there's a display name field which makes this confusing